### ISSUE TITLE
Cleaning or Spot-Cleaning didn't update the Homey-Tag

### ISSUE THREAD

**Issue by CyberSponk** [Predjee/com.irobot.roomba980#4]
Homey 5.0.0-rc.34
iRobot-Homey-App 3.0.0
iRobot Vacuum-Cleaner i7(+)

When the Vacuum-Cleaner is running (Started through Rommba-Company-App) the Tag on Homey will not be updated. It shows "docked" and not cleaning. During this a Flow with "When > State changes to Cleaning" and also "When > State changes to Spot-Cleaning" will not executed.

The Battery-State is updating during cleaning-time without Problems or Issues.

**Comment by Predjee** [comment]
I will have a look into this issue. It seems multiple roomba's have this same issue.

**Comment by CyberSponk** [comment]
It seems so because same with Roomba Braava M6.

**Comment by Predjee** [comment]
Can you try out the new just released SDK 3 (Homey v5 supported) version. If you don't have Homey v5 then this won't work since I completely refactored for the new SDK aswell. 

The version is available on https://homey.app/nl-nl/app/com.irobot.roomba980/iRobot/test/

**Comment by CyberSponk** [comment]
Nope, the same as before updating the App > No State-Update when cleaning. Only shows "Docked" for both Devices.

**Comment by Predjee** [comment]
I just released a new test version. I made a few stupid mistakes :) The Braava doesn't give back the correct state just yet. The mapping of data is completely different. Roomba should send an update with the new test version!

**Comment by CyberSponk** [comment]
Tested it with 4.0.2 Beta > Same as before, no Status-Update when cleaning.

**Comment by CyberSponk** [comment]
Made a new Test-Flow: "When > State changes to Cleaning, Then send Push "Cleaning" " with "Tag" of the iRobot.

When starting Cleaning trough Roomba-App i got a Push with "Cleaning docked"

So: With 4.0.2 the Flow will be executed (thats new) bit the Tag on Homey will not be updated.
